modules/imgcodecs/src/grfmt_spng.hpp

```diff
@@ -51,6 +51,12 @@ class SPngEncoder CV_FINAL : public BaseImageEncoder
 
 protected:
     static int writeDataToBuf(void *ctx, void *user, void *dst_src, size_t length);
+
+private:
+    int m_compression_level;
```

Comment:
Those do not need to be members right? They are only used in one function

Reply:
used in PngEncoder::write() and PngEncoder::writeanimation()